GSA/made-in-america-data
# [MIA-170,168,251,263, and 249]: Code Review Contribs

needs to be considered and merged before https://github.com/GSA/made-in-america-data/tree/stage can be merged into https://github.com/GSA/made-in-america-data/tree/develop

Description
after a review of the code on `stage` branch, the following issues were apparent and this code provides solutions for them.

issues included:
- eslint engine not included
- unused static methods
- zombie code
- lint ignore rules with page-level overrides does not include a comment
- declarations not being used
- unused files existed in tree
- eslint not configured for mocha or chai
- test directory was being lint ignored
- some code unnecessarily eslint ignored
- variable casing not following best practices or consistent

How to test
- ensure code is sync'd with latest from `stage`
- review code
- execute ` node buildwaiverdata.js`
- execute ` node buildcsvdata.js`
- execute ` npm run test`

To prepare for code review, please follow the following checks.

 - [ ] pipeline link updated in ticket
